Makefile

```diff
@@ -38,8 +39,8 @@ fmt:
 	@go fmt -x "$(REPO)/..."
 
 image: build
-	cd docker && docker build -f Dockerfile.contacts -t $(DOCKER)-contacts:latest .
-	cd docker && docker build -f Dockerfile.gateway -t $(DOCKER)-gateway:latest .
+	docker build -f docker/Dockerfile.contacts -t $(DOCKER)-contacts:latest .
```

Comment:
I think this modification changes the context that gets passed into the docker process. Was this intentional? From the look of the docker files, this change should make it actually work.

Reply:
As I recall, I couldn't build functional images because being in /docker the build couldn't see the /bin directory to copy into the containers. There might be other ways to fix that, but this seemed to work.